charts/graphscope-store/values.yaml

```diff
@@ -81,9 +81,14 @@ executorQueryStoreThreadCount: 2
 log4rsConfig: "LOG4RS_CONFIG"
 
 
-
+## Key-value pair seperated by ^
+## For example extraConfig=k1=v1^k2=v2
 extraConfig: ""
 
+auth:
+  username: ""
+  password: ""
```

Comment:
Could we use a k8s secret (which should be more standard https://kubernetes.io/docs/concepts/configuration/secret/)? Rather than plain username and password?

We could ask user to create a secret first and use the secret to install our helm charts.

Reply:
This will be enhanced by another PR~